docs/whats-new/dotnet-docs-2021-07-01.md

```diff
@@ -117,7 +117,7 @@ The following people contributed to the .NET docs during this period. Thank you!
 - [Der-Kraken](https://github.com/Der-Kraken) (1)
 - [eriawan](https://github.com/eriawan) - Eriawan Kusumawardhono (1)
 - [erri120](https://github.com/erri120) - erri120 (1)
-- [github-actions](https://github.com/github-actions) -  (1)
+- github-actions -  (1)
```

Comment:
I've got this on my list to fix for next month. Thanks for making the change here.